runner/repl.js

```diff
@@ -79,6 +79,15 @@ const memoryToString = mem => {
 };
 
 let prev = '';
+
+function setLastEvalExp(prev, ret) {
+  const lastEvalExp = `let _ = ${ret != null ? ret : "undefined"};\n`;
+  if (/let _ = .*;\n/.test(prev)) {
```

Comment:
Won't this break after subsequent runs? If we're going with this approach it should be `var` not `let`.

Reply:
I don’t understand what you are saying.

are you referring to redeclaring the let statement? In this case, we only declare it once for each sequence (just change a assign expression)

Reply:
Can't you just do `_ =` instead of having to deal with let/var and check every time/etc?

Reply:
hmmm, makes sense i’ve made the change...